tests/jvm/src/test/scala/scala/meta/tests/cli/CliSuite.scala

```diff
@@ -0,0 +1,39 @@
+package scala.meta.tests.cli
```

Comment:
@olafurpg Since this was necessary for the "Overview" section of the specification, I've finally upstreamed my personal helper tools: `metac` (compile Scala files into SemanticDB) and `metap` (decompile SemanticDB files). See this test for an example of how these tools work.

Reply:
🎉